server/samples/metastore.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE IF NOT EXISTS asherah;
```

Comment:
we have the same DDL in `Metastore.md`. Can we somehow point at the same reference for both the places? This would help in removing redundancy.

Reply:
This file is used by the `docker-compose-mysql.yaml` sample to init the mysql db and needs to be standalone.